<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>group_chat</class>
 <widget class="QWidget" name="group_chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>聊天室</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>在线用户：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pre_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>384</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>450</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>450</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_Browser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>301</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="p_textBrowser">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>191</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
